src/xdsl/xdsl_opt_main.py

```diff
@@ -137,7 +138,7 @@ def register_all_arguments(self, arg_parser: argparse.ArgumentParser):
     def register_all_dialects(self):
         """
         Register all dialects that can be used.
-
+        
```

Comment:
Could you remove the trailing spaces?

Reply:
I think you might have resolved it by mistake, it's still here ^^
If you want I can remove it then merge the PR, but I don't want to complexify your git rebase!

Reply:
Oh yes, thanks for catching that. I attempted a rebase but one of the test fails now. We can look at it together tom and i can resolve this as well then.